packages/flutter/lib/src/material/page.dart

```diff
@@ -5,12 +5,19 @@
 import 'dart:async';
 
 import 'package:flutter/widgets.dart';
+import 'material.dart';
+import 'theme.dart';
 
 final FractionalOffsetTween _kMaterialPageTransitionTween = new FractionalOffsetTween(
   begin: FractionalOffset.bottomLeft,
   end: FractionalOffset.topLeft
 );
 
+final FractionalOffsetTween _kIosPageTransitionTween = new FractionalOffsetTween(
```

Comment:
for privates, use "Cupertino" and "MountainView" instead of "iOS" and "Android" so that the capitalisation of "iOS" doesn't cause problems (like it does here, with "Ios")